CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   See [PR #257](https://github.com/allenai/allennlp-models/pull/257) for more details.
 - Added a `beam_search` parameter to the `generation` models so that a `BeamSearch` object can be specified in their configs.
 - Added a binary gender bias-mitigated RoBERTa model for SNLI.
+- Added `StanfordSentimentTreeBankDatasetReader.apply_token_indexers()` to add token_indexers rather than in `text_to_instance` 
```

Comment:
Can you move this up to line 13, under "Unreleased"?

Reply:
ok